jest.test.config.js

```diff
@@ -1,5 +1,42 @@
+// Resolve the absolute path of the caller location.
+const rootPath = process.cwd();
+
 module.exports = {
-  preset: '@commercetools-frontend/jest-preset-mc-app',
+  displayName: 'test',
+  globals: {
+    'process.env': {
+      NODE_ENV: 'test',
+    },
+  },
   moduleDirectories: ['src', 'node_modules'],
+  moduleNameMapper: {
+    '\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$':
+      '<rootDir>/test/transform-file.js',
+    '\\.css$': 'identity-obj-proxy',
+  },
+  rootDir: rootPath,
+  setupFiles: [
+    'raf/polyfill',
+    '<rootDir>/test/setup-tests.js',
+    'jest-localstorage-mock',
+  ],
   setupTestFrameworkScriptFile: '<rootDir>/scripts/setup-test-framework.js',
+  snapshotSerializers: ['enzyme-to-json/serializer'],
+  testEnvironment: 'jsdom',
+  testURL: 'https://mc.commercetools.com/',
```

Comment:
do we need this?

Reply:
We don't need this exact value but it's good to define a value for it. It sets the value you get in JSDOM when reading `window.location.href` for example.